bin/zuul

```diff
@@ -333,4 +312,42 @@ scout_browser(function(err, all_browsers) {
     });
 });
 
+function readLocalConfig(config) {
+    var yaml = path.join(process.cwd(), '.zuul.yml')
+    var js = path.join(process.cwd(), 'zuul.config.js')
+    var yamlExists = fs.existsSync(yaml)
+    var jsExists = fs.existsSync(js)
+    if (yamlExists && jsExists) {
+        console.error('both .zuul.yaml and zuul.config.js are found in the project directory')
+        process.exit(1)
+    } else if (yamlExists) {
+        return mergeConfig(config, readYAMLConfig(yaml))
+    } else if (jsExists) {
+        return mergeConfig(config, require(js))
+    } else {
+        return config
+    }
+}
+
+function readGlobalConfig(config) {
+    var filename = find_nearest_file('.zuulrc') || path.join(osenv.home(), '.zuulrc')
```

Comment:
Good point... what do you think should be the name of file within the home directory? The same `.zuulrc`? We could try to parse it with YAML and then fallback to JS.